proposals/drafts/campaign_finance_filings.rst

```diff
@@ -0,0 +1,425 @@
+funders, expenditures, organizational ties and other financial details.
+Filings encapsulate some number of possible specific disclosures, and some
+filings can contain more than one type of disclosure, so this proposed option is
+to consider each actual filing as an object and then each specific disclosed
+piece of information as associated with that object. So a CommitteeNameChange
+would be associated with a Filing, along with a CommitteeAddressChange and a
+CommitteeOfficerAddition. A series of CommitteeContributions and
+CommitteeExpenses would be attached to one Filing. This way we don't need to
+model the specific disclosure forms in each state, and just directly extract
+the most meaningful parts of the filing.
+
+We want to make it easy for people to do manipulations on top of this data,
+while ensuring that this system itself does as little manipulation as possible
+to the underlying data from reports filed. For instance, many records include
+Person entities - these are deterministically extracted from just the available
+filing data, e.g. an address, employer, occupation and name are all extracted
+for each contribution. Deduplication can then happen with each of these
+entities, but they are themselves totally reproducible from just the filing
+information.
+
+This model entails that Filings have Sections - a Filing can, and often will,
+have multiple Sections. As much of the specific logic as possible should be
+pushed down into Sections, which offer flexibility by allowing us to compose
+them together to reproduce the data from state-specific reports. We can model
+a particular state filing form as a Filing with a specific set of expected
+Sections to be present. Some attributes that might make sense as properties of a
+Filing, such as Amendments, are here presented as Sections; the line between the
+two is muddy, but this proposal tries to keep Filings as minimal as possible.
+
+Finally, the different entity types presented here present either current or
+likely future conflicts with other entities in the Open Civic Data namespace, so
+this proposal doesn't make any attempt to resolve those conflicts since they
+seem ancillary for now.
+
+
+Implementation
+==============
+
+Filing
+------
+
+id
+    Open Civic Data-style id in the format ``ocd-cf-filing/{{uuid}}``
+
+filing_type
+    **optional**
+    FilingType (jurisdiction-specific)
+
+filing_committee
+    Committee
+
+filing_date
+    Date (and possibly time) when filing was submitted.
+
+filing_coverage_begin_date
+    **optional**
+    Date (and possibly time) when filing period of coverage begins.
+
+filing_coverage_end_date
+    **optional**
+    Date (and possibly time) when filing period of coverage ends.
+
+filing_regulator
+    Regulator
+
+filing_url
+    **optional**
+    Wish it wasn't optional.
+
+filing_relevant_election_date
+    Date of (nearest? next?) relevant election.
+
+filing_person
+    **optional**
+    Person responsible for the filing.
+
+Committee
```

Comment:
If I'm reading the [FEC committee spec](https://api.open.fec.gov/developers/#!/committee/get_committees) provided by @LindsayYoung properly, I believe the issue you raise @jsfenfen is addressed there by first classifying the committees with its the "committee type" (e.g. Independent expenditure, Candidate) and then recording details about its relationship to candidates via its "designation" (e.g. Belonging to candidate, Authorized by candidate, Joint fundraising, etc.). Maybe that's something we should here as well?